plugins/Marketplace/Plugins.php

```diff
@@ -296,6 +298,9 @@ private function enrichPluginInformation($plugin)
             }
         }
 
+        $plugin['licenseStatus'] = $consumer[$plugin['name']]['licenseStatus'] ?? '';
+        $plugin['hasDownloadLink'] = !empty($plugin['versions'][0]['download']);
```

Comment:
Not sure if it's correct to use the first version here, is it the latest one?

Reply:
Yaa seeing the marketplace code, it does look like we get the latest version

![Uploading Screenshot from 2024-09-09 14-09-04.png…]()